python/pyspark/pandas/generic.py

```diff
@@ -3064,25 +3064,25 @@ def ffill(self, axis=None, inplace=False, limit=None) -> Union["DataFrame", "Ser
 
     @property
     def at(self) -> AtIndexer:
-        return AtIndexer(self)
+        return AtIndexer(self)  # type: ignore
```

Comment:
Do we need the `ignore`? What happens without it?

Reply:
error: Argument 1 to "AtIndexer" has incompatible type "Frame"; expected "Union[Series[Any], DataFrame[Any]]"